test/ts-scenario/steps/lib/channel.ts

```diff
@@ -163,6 +166,7 @@ export async function cli_channel_update(channelName: string, updateTx: string,
 			BaseUtils.logMsg(`Channel ${channelName} has been updated`);
 		}
 	} catch (err) {
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
At a glance not sure the best change to make but this seems like a code change should be able to remove the lint error